CHANGELOG.md

```diff
@@ -36,6 +36,7 @@
         - Fix removal of cached opengraph photos.
         - Do not email inactive body comment users. #3587
         - Look up organizational domain in DMARC checking. #3603
+        - Stop slash in category name breaking csv download #2679
```

Comment:
This is a public changelog so we can't reference issues in the SocietyWorks repo. The references are actually to pull requests in this repo. So this needs changing to `#3642`.